### PR TITLE
Move public winningBids to auction

### PR DESCRIPTION
**Pull request by jaiminpanchal27**
## Type of change
- [x] Feature

## Description of change
Move _winningBids to auction.

### CODE REVIEW

File: src/auctionManager.js [hunk 1]
```diff
@@ -31,6 +31,20 @@ export function newAuctionManager() {
   let _auctions = [];
   let _public = {};
 
+  _public.addWinningBid = function(bid) {
+    let auction = _auctions.find(auction => auction.getAuctionId() === bid.auctionId);
```
Comment (matthewlane) [inline src/auctionManager.js]: This could be a `const`

File: src/secureCreatives.js [hunk 2]
```diff
@@ -31,7 +32,7 @@ function receiveMessage(ev) {
       sendAdToCreative(adObject, data.adServerDomain, ev.source);
 
       // save winning bids
-      $$PREBID_GLOBAL$$._winningBids.push(adObject);
+      auctionManager.addWinningBid(adObject);
```
Comment (matthewlane) [inline src/secureCreatives.js]: There's one last `_winningBids` down on line 47

Comment (jaiminpanchal27) [reply src/secureCreatives.js]: Removed.

**Comment by mkendall07** [comment]
image

File: src/auctionManager.js [hunk 3]
```diff
@@ -29,9 +29,12 @@ const CONSTANTS = require('./constants.json');
  */
 export function newAuctionManager() {
   let _auctions = [];
-  let _public = {};
+  let auctionManager = {};
```
Comment (mkendall07) [inline src/auctionManager.js]: 💯

**Pull request merged**